main.scss

```diff
@@ -157,3 +167,90 @@
 .invite-colleagues__share-text {
 	padding: 10px 12px 10px 0;
 }
+
+.invite-colleagues-banner__wrapper {
+	@include oBanner($class: 'o-banner', $themes: 'marketing-secondary');
+
+	.invite-colleagues__copy-link-button {
+		@include oButtonsTheme(inverse);
+		background-color: getColor('black');
+		color: getColor('white');
+		border: getColor('black');
+		@include oGridRespondTo($until: M) {
+			margin-bottom: 12px;
+		}
+	}
+
+	.o-banner__inner.invite-colleagues-banner {
+		display: flex;
+		flex-wrap: wrap;
+	}
+
+	//<--- TODO: this is here because of origami styling issue on o-banner - remove when issue resolved
+	& .invite-colleagues__copy-outer {
+		margin: 12px 0;
+		@include oGridRespondTo(L) {
+			margin: 0;
+		}
+	}
+
+	& .invite-colleagues__copy-input {
+		margin-bottom: 12px;
+		@include oGridRespondTo(M) {
+			margin: 0;
+		}
+	}
+
+	& .invite-colleagues__share {
+		margin-top: 12px;
+		@include oGridRespondTo(L) {
+			margin: 0;
+		}
+	} //--->
+
+
+	.o-banner__content {
+
+		&.invite-colleagues-banner__intro {
+			flex-basis: 100%;
+			text-align: center;
+			@include oGridRespondTo(L) {
+				flex: 2;
+				text-align: left;
+			}
+		}
+
+		&.invite-colleagues-banner__copy {
+			flex-basis: 100%;
+			@include oGridRespondTo(L) {
+				flex: 2;
+				padding-right: 0;
+			}
+		}
+
+		&.invite-colleagues-banner__share {
+			flex-basis: 100%;
+			@include oGridRespondTo(L) {
+				flex: 1;
+				padding-left: 0;
+			}
+		}
+
+		& ul {
```

Comment:
Is all this necessary to override something due to specificity?

Also, the preceding `& ` is superfluous.